features/collections.feature

```diff
@@ -1,5 +1,30 @@
 Feature: Collections
 
+  Scenario: Automatic opt-in to the B variant from navigation pages without any
+    previous variant
+    Given I do not have any AB testing cookies set
+    When I visit "/education"
+    Then I am in the "B" variant of the education navigation test
+    And I stay on the same bucket when I keep visiting "/education"
+
+  Scenario: Automatic opt-in to the B variant from navigation pages with
+    variant A
+    Given I am in the "A" group for "EducationNavigation" AB testing
+    When I visit "/education"
+    Then I am in the "B" variant of the education navigation test
+    And I stay on the same bucket when I keep visiting "/education"
+
+    When I visit "/education/running-and-managing-a-school"
+    Then I am in the "B" variant of the education navigation test
+    And I stay on the same bucket when I keep visiting "/education/running-and-managing-a-school"
+
+  Scenario: Don't automatically opt-in to B variant on content pages
+    that start with "education"
+    Given I am in the "A" group for "EducationNavigation" AB testing
+    When I visit "/education-maintenance-allowance-ema"
+    Then I am in the "A" variant of the education navigation test
+    And I stay on the same bucket when I keep visiting "/education"
```

Comment:
Should this path match line 24? Otherwise I'm guessing this step will fail because visiting /education will put you in B.